.github/workflows/ci.yml

```diff
@@ -147,6 +147,7 @@ jobs:
         env:
           DEVHUBDB_PAT: ${{ secrets.DEVHUBDB_PAT }}
           NYRKIO_TOKEN: ${{ secrets.NYRKIO_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Hm, shouldn't this literally be ` GH_TOKEN: ${{ github.token }}`, such that its a read-only token?

The `secrets.GITHUB_TOKEN` can public a new release, and that's not something we want to be able do from within devhub

Reply:
So `github.token` and `secrets.GITHUB_TOKEN` are [identical](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret) according to GitHub.

I think we should change `GITHUB_TOKEN` to be [default restrictive](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#setting-the-permissions-of-the-github_token-for-your-repository), and then add back permissions for publish.

Reply:
Oh, excellent catch, thanks!